### PR TITLE
Remove reset suggestions from update failures.

### LINKED ISSUE #770: Failed auto-update prompts for reset
[Per some issues](http://new-xkit-extension.tumblr.com/post/133312389809/server-outage-and-xkit-reset-popup) with GitHub pages at [~ 21:51 EST](https://status.github.com/messages/2015-11-16), a number of users reported being prompted to reset XKit to fix the issue.  This is _shockingly_ poor UX, even by XKit standards.  We should not be prompting the user to reset except in the most dire of circumstances, usually after they've reached out to support.

The relevant line(s) are in [`xkit_updates.js`](https://github.com/new-xkit/XKit/blob/5e6976317ba8835c58bc923e114854676ae0e6ca/Extensions/xkit_updates.js#L135).  Resetting XKit [deletes all values in the storage](https://github.com/new-xkit/XKit/blob/5e6976317ba8835c58bc923e114854676ae0e6ca/xkit.js#L986-L991), with no recourse or undo.

### PR DESCRIPTION
**Pull request by nightpool**
Remove reset suggestions from update failures.

Keep reset suggestion for when xkit_updates is missing.

Fixes #770

### CODE REVIEW

**Comment by 0xazure** [comment]
We need version increments on these, do we not?

**Comment by nightpool** [comment]
Whoops! will fix.

File: Extensions/xkit_preferences.js [hunk 1]
```diff
@@ -1405,33 +1405,41 @@ XKit.extensions.xkit_preferences = new Object({
 
 			$("#xkit-extensions-panel-right-inner").html('<div id="xkit-extension-panel-no-settings">Updating...</div>');
 
-			if (typeof XKit.extensions.xkit_updates === "undefined" || typeof XKit.extensions.xkit_updates.update === "undefined") {
+			if (typeof XKit.extensions.xkit_updates === "undefined" ||
+				typeof XKit.extensions.xkit_updates.update === "undefined") {
+
 				XKit.window.show("Can't update",
-					'It looks like "XKit Updates" extension is missing or not working properly. It is highly recommended that you reset XKit.', "error",
+					'It looks like "XKit Updates" extension is missing or not working properly. '+
```
Comment (0xazure) [inline Extensions/xkit_preferences.js]: Can we tweak the wording here?  `XKit Updates` is an internal extension that isn't visible through the gallery, I don't think this message would make sense to a user.  Maybe something more general, like "an internal part of XKit is missing or not working properly"?

Comment (nightpool) [inline Extensions/xkit_preferences.js]: scope creep, tbh. all of our copy/messaging needs a good going over

Comment (0xazure) [inline Extensions/xkit_preferences.js]: Right, but we're touching these lines anyways.

I definitely agree though, we need to run through our copy all at once so we're not duplicating work.

**Comment by hobinjk** [comment]
Functionality LGTM

**Comment by hobinjk** [comment]
Which is to say that I would grant review if there were version bumps

**Comment by hobinjk** [comment]
@nightpool

**Comment by nightpool** [comment]
Don't have time to address this now but hopefully will later this week

**Comment by nightpool** [comment]
superseded by #1185
